src/components/EventListCard/EventListCard.tsx

```diff
@@ -38,6 +38,8 @@ function eventListCard(props: InterfaceEventListCardProps): JSX.Element {
   const [recurringchecked, setRecurringChecked] = useState(false);
   const [publicchecked, setPublicChecked] = useState(true);
   const [registrablechecked, setRegistrableChecked] = React.useState(false);
+  const [eventDeleteisOpen, setEventDeleteModalIsOpen] = useState(false);
```

Comment:
Follow the naming convention as 
```js
const [state,setState] = useState();
```

`eventDeleteisOpen` var should be, eventDeleteModalIsOpen, follow the same for `eventUpdateIsOpen` var.